README.md

```diff
@@ -103,7 +103,19 @@ $ docker run -d \
 
 * Then start any containers to be proxied as described previously.
 
-Note: If the docker-gen container name is static and you want to explicitly set it, use `-e NGINX_DOCKER_GEN_CONTAINER=nginx-gen`. The same thing is true with the nginx container (`-e NGINX_PROXY_CONTAINER=nginx`).
```

Comment:
I've removed the -e switch from the variable names.

Reply:
see below